tasks/cleanup-workspace/tests/test-cleanup-workspace-invalid-subdirectory.yaml

```diff
@@ -16,6 +16,8 @@ spec:
       params:
         - name: subdirectory
           value: "nonexisting"
+        - name: delay
+          value: "1s"
```

Comment:
In general, tests should never add unnecessary delays when executing. I guess in this case it's less of an issue compared to, say, unit tests in Python, where adding a 1s delay may double the total time of test exection. But I think it's still nice to avoid delaying test execution if possible.

So my first thought was this should be `"0s"`. But in fact, I think the `sleep` command should be mocked and in at least the main passing test pipeline, we should check that sleep was called once with the right parameter.

Reply:
mocked `sleep`